### PR TITLE
Merge org.librehunt.Organizer (Organizing app)

### PR DESCRIPTION
**Pull request by aviwad**
I'll try this again! I closed the previous pull request and changed almost everything required, besides the indentation problem, which I'm confused with as it looks fine (and similar to other json files in flathub). I hope this gets accepted now!
#861

### CODE REVIEW

**Comment by aviwad** [comment]
bot, build org.librehunt.Organizer
(do I call bot or testbot?)

**Comment by aviwad** [comment]
why isn't the bot building? Could someone help? Did I call it wrong?

**Comment by aviwad** [comment]
testbot, build org.librehunt.Organizer

**Comment by aviwad** [comment]
ok, newer issue: opening the flatpakref file says the "screenshots can't be downloaded", while if I manually open the link, the images download. Is there something wrong with the images?

**Comment by aviwad** [comment]
bot, build org.librehunt.Organizer
testbot, build org.librehunt.Organizer

**Comment by aviwad** [comment]
the screenshots 404'ing is still an issue. is this related to a bug in my code or an error with Gnome Software?

**Comment by aviwad** [comment]
temporarily removing the commit version and testing to see if the appdata glitch occurs
bot, build org.librehunt.Organizer
testbot, build org.librehunt.Organizer

**Comment by aviwad** [comment]
attempt to fix screenshot 2
bot, build org.librehunt.Organizer
testbot, build org.librehunt.Organizer

**Comment by aviwad** [comment]
attempt 3, by checking if Lollypop's screenshots would work.
testbot, build org.librehunt.Organizer

**Comment by aviwad** [comment]
Even org.gnome.Lollypop's screenshots error out here. I give up. @nedrichards should I revert my app back to the stable version and let my app come in without functioning screenshots? Is there a way you could help me debug why no screenshots work in my appdata?

**Comment by aviwad** [comment]
one last try before I lose hope
testbot, build org.librehunt.Organizer

**Comment by alexlarsson** [comment]
bot, build org.librehunt.Organizer

**Comment by alexlarsson** [comment]
Test build here: https://flathub.org/builds/#/builders/2/builds/8146

**Comment by nedrichards** [comment]
On the indenting, look at the libhandy section in the 'files changed' view here. Looks like you have one tab too many for the URL.

also fwiw you can run testbot yourself on PRs (that only tests x86) flathub admins can run the main bot which builds on all architectures. When the app is accepted you can run both. This will probably change a little in future.

**Comment by aviwad** [comment]
I see! Is the indenting better? I just saw the error on the URL element of libhandy, everything else seems ok...

File: org.librehunt.Organizer.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "command" : "organizer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro"
+    ],
+    "build-options" : {
```
Comment (bilelmoussaoui) [inline org.librehunt.Organizer.json]: Those build-options are now used by default on GNOME Sdk 3.30 (which is based on Freedesktop Sdk 18.08)

Comment (aviwad) [reply org.librehunt.Organizer.json]: these?
```
    "build-options" : {
        "cflags" : "-O2 -g",
        "cxxflags" : "-O2 -g",
        "env" : {
            "V" : "1"
        }
    }
```

**Comment by aviwad** [comment]
all clear? :D

File: org.librehunt.Organizer.json (hunk 1 above)
Comment (bilelmoussaoui) [reply org.librehunt.Organizer.json]: Yes

Comment (aviwad) [reply org.librehunt.Organizer.json]: well, I fixed that now, so is this good to merge? :D

**Comment by nedrichards** [comment]
bot, build org.librehunt.Organizer
testbot, build org.librehunt.Organizer

**Comment by aviwad** [comment]
Woke up to this in midnight, so I can't test the flatpak built, but I assume it works perfectly?

**Comment by aviwad** [comment]
it's the same thing, but upgrading to the newer commit without the "testing123" in the summary

**Comment by aviwad** [comment]
just to make sure
testbot, build org.librehunt.Organizer

**Comment by aviwad** [comment]
awesome, it works.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.librehunt.Organizer

Thanks!

**Comment by aviwad** [comment]
YAY THANKS <3

**Pull request closed without merging**